fs/accounting/accounting.go

```diff
@@ -47,7 +113,8 @@ const averagePeriod = 16 // period to do exponentially weighted averages over
 
 // NewAccountSizeName makes a Account reader for an io.ReadCloser of
 // the given size and name
-func NewAccountSizeName(in io.ReadCloser, size int64, name string) *Account {
+func NewAccountSizeName(ctx context.Context, in io.ReadCloser, size int64, name string) *Account {
```

Comment:
This function is only used inside the package maybe we should not export it to avoid confusion with `Stats.NewAccount` function.